packages/apps/contracts/TicTacToeApp.sol

```diff
@@ -223,4 +194,12 @@ contract TicTacToeApp is CounterfactualApp {
     }
   }
 
+  function resolveType()
```

Comment:
I haven't seen this used in `HighRollerApp.sol`, why not?

Reply:
It's used in the client-side code during uninstall, but never used in solidity